libc/include/llvm-libc-macros/limits-macros.h

```diff
@@ -69,8 +69,8 @@
 #define LONG_WIDTH __LONG_WIDTH__
 #elif defined(__WORDSIZE)
 #define LONG_WIDTH __WORDSIZE
-#else
-#error "Unknown WORDSIZE to define LONG_WIDTH."
+#else // use sizeof(long) * CHAR_BIT as backup.
+#define LONG_WIDTH (sizeof(long) * CHAR_BIT)
```

Comment:
Consider adding a comment that this is necessary for clang-11 (and any other clang versions; use godbolt.org to test quickly)

Reply:
clang-14 is the first release to define `__LONG_WIDTH__`.

Reply:
ALSO RIP platforms where CHAR_BIT != 8. lol